### ISSUE TITLE
Location as variable

### ISSUE THREAD

**Issue by Toucan-Sam** [RestingCoder/discord-widget#22]
**TL:DR;** Widget doesn't support HTTPS, should be modified for self hosting/HTTPS by adding the script location as a variable.

**Long:** Josh, thank you for hosting the widget of your own volition and with your own resources. I've had to modify it slightly to work with my own site as it's served only via HTTPS and so non-HTTPS things make it unhappy. In the short term I just modified the locations directly. In the long term I think it would be more effective to allow for the script location to be a variable (set to your http://discord.deliriousdrunkards.com by default) so that if someone wishes (like me) to host it themselves/on HTTPS, they only need to change the location once. :)

If I get around to fixing it properly instead of my short term fix, I'll submit a pull request.

Thanks again!

**Comment by RestingCoder** [comment]
Nothing is stopping anyone from just downloading the script file (normal or minified) and self-hosting it. The only thing connected to my server at that point would be the CSS, which can also be self-hosted.
